Clippit/Word/DocumentAssembler.cs

```diff
@@ -1003,7 +965,7 @@ OpenXmlPart part
             // assign unique image and paragraph ids. Image id is document property Id  (wp:docPr)
             // and relationship id is rId. Their numbering is different.
             const string imageId = InvalidImageId; // Ids will be replaced with real ones later, after transform is done
-            var relationshipId = GetNextImageRelationshipId(part);
+            var relationshipId = Relationships.GetNewRelationshipId();
```

Comment:
@MalcolmJohnston just wondering why you decided to use random generated id instead of deterministic/sequential one?